src/js/tech/tech.js

```diff
@@ -238,6 +238,14 @@ class Tech extends Component {
    * @method dispose
    */
   dispose() {
+    // clear out text tracks because we can't reuse them between techs
```

Comment:
No, we cannot maintain them. If we're switching from Html to Flash, we're moving from track elements to emulated text tracks. And then if if we're going from Flash back to Html, we would want to clear out our emulated text tracks and use the native track elements.

If you're just switching sources but staying in the same tech, you should keep the tracks.